### ISSUE TITLE
Add support for ReplayGain 2.0 pluggin

### ISSUE THREAD

**Issue by techie2000** [mikenye/docker-picard#63]
This plugin supersedes the ReplayGain plugin, but  to use it, requires rsgain to be available ([https://github.com/complexlogic/rsgain](https://github.com/complexlogic/rsgain)). Can this be included in the image in a future release?

**Comment by techie2000** [comment]
Just seen that this is a duplicate of #61. 
I'll leave it open until you have read it as it contains link to the dependency which the original issue does not.

**Comment by dchawisher** [comment]
I would also find this very helpful. Failing this, I will have to use a convoluted workflow with Beets

**Comment by thepebbler** [comment]
Another vote.. the only thing that is missing for my perfect workflow is replaygain being applied to ensure all music players I use are able to normalise the audio.

**Comment by gozes** [comment]
yup more votes from me. only reason I can't deploy this to my nas and give up my NFS base workflow is because its missing replaygain 2.0. so hopping this will get added

**Comment by techie2000** [comment]
Hi @mikenye I can see you created a new branch for this issue https://github.com/mikenye/docker-picard/tree/rsgain

Is there anything you are aware of that would stop this being merged to master/prod?

**Comment by sikyist** [comment]
im sorry if this is not the right place, but I cannot find anyway/anywhere else to ask. 
how do i get rsgain working on this docker? where do I direct picard to for rsgain?

**Comment by minchingtonak** [comment]
At this point it seems that the necessary changes have been merged, but a new release still needs to be published to Dockerhub. @mikenye could you help out with this?

edit: on further investigation it seems the image build is failing and blocking the release of a new image version. I'll take a look when I have the time

**Comment by PaulMcF1987** [comment]
Is there any update on this?
I have the latest docker image installed but replaygain 2.0 doesnt work

It appears that this image is created for ubuntu 18.xx and I am on ubuntu 22.xx and it installs `jammy` packages which seem to be too new for the image?
